doc/data/messages/m/missing-final-newline/bad.py

```diff
@@ -0,0 +1,9 @@
+# using LF
+
+eat("apple", "candy")  # \n
+print(123)  # EOF
+
+# using CRLF
+
+eat("apple", "candy")  # \r\n
+print(123)  # EOF
```

Comment:
```suggestion
print(123)  # EOF
# +1: [missing-final-newline]
```

Reply:
Thanks! It worked without a line offset.